app/assets/stylesheets/registration.scss

```diff
@@ -1,54 +1,79 @@
 .page-registrations {
-  .ball {
-    background: image-url('branding/ball.png') no-repeat;
-    background-size: contain;
-    height: 633px;
-    max-width: 100%;
+  // For the mobile version which doesn't have the same global CSS
+  background: $body-bg;
+
+  #app {
+    margin: -10px;
+    margin-bottom: 10px;
   }
+  // End specific mobile
 
-  .v-center {
-    display: table;
-    height: 633px;
+  // Overriding bootstrap
+  .form-control {
+    display: inline-block;
+    width: 320px;
   }
+  // End Overriding bootstrap
 
-  @media (max-width: $screen-xs-max) {
-    .v-center {
-      height: auto;
+  .fields {
+    margin: $triple-margin 0;
+
+    @media screen and (max-width: $breakpoint) {
+      flex-direction: column-reverse;
+      margin: $double-margin 0;
+    }
+
+    section {
+      border-left: solid 3px $brand-primary;
+      padding-left: 10px;
     }
   }
 
-  .content {
-    display: table-cell;
-    vertical-align: middle;
+  .advice {
+    max-width: 450px;
+  }
+
+  .captcha {
+    align-items: center;
+    display: flex;
 
-    h1 {
-      font-size: 35px;
-      margin: 12px 0;
+    .captcha-img {
+      margin-left: 5px;
+      order: 2;
```

Comment:
I don't know if this is intended, but this only works in the "desktop" (default) view, but not in "mobile" view. Since both should use the same responsive design, I assume this should look always the same on small screens, independent of in which mode you actually are? So you probably just want to delete `app/views/simple_captcha/_simple_captcha.mobile.haml` so it uses the same template as in "desktop" view?